docs/wiki/introduction/sql.md

```diff
@@ -369,6 +369,35 @@ The following trig functions: `sin`, `cos`, `tan`, `cot`, `asin`, `acos`, `atan`
     </p>
     </details>
 
+- `version_compare(VERSION_STRING, COMPARE_OPERATOR, VERSION_STRING, EPOCH, DELIMITER_PRECEDENCE, COMPARE_REMAINING, REMAINING_PRECEDENCE)`: return `1` if comparison is truthy or `0` for falsy. compare_operator must be one of the following: (`<`, `<=`, `=`, `>=`, `>`). epoch, delimiter_precedence, compare_remaining, and remaining_precedence are all optional booleans that change the behavior of the comparison, which are mostly used for linux package versioning.
```

Comment:
I think this is too much complexity exposed. I think we should expose something very simple -- `compare_version(VERSION_STRING, VERSION_STRING)` and it should return -1, 0, 1. With the usual semantics. 

if we want, we could  expose it with an optional 3rd argument for what flavor of version.

Reply:
I agree and have made those changes. Does the new definition look good?